README.md

```diff
@@ -82,6 +67,10 @@ require a unique way of handling them, respectively.
    but also an "inverted" policy which matches all shoots that don't have ACL
    enabled. All these policies are then put in a single EnvoyFilter patch.
 
+![Listener Overview](./docs/listener-overview.svg)
+
+*Please note that the `Kubernetes Service Listener` doesn't exist anymore in current versions of Gardener.*
+
```

Comment:
Let's drop the diagram instead of keeping an outdated version. It doesn't provide much benefit FMPOV.